File: program/fizzbuzz.py


##Pythonでのfizzbuzzです
##if文の中身はそれぞれ"0と等しいか","0かそれ以外か判定して0",
##  "0かそれ以外かを判定した後0であった場合次の処理を行う"処理です

##This is fizzbuzz written in Python
##"if" in this program means "Is it equal to 0?",
#  "Check if it is 0 or not and then it equal to 0 go next process",
#  and "After checking if it is 0 or not. 
# If it is 0, do the next process".

for i in range(1,100):
    if((i%15)==0):
        print("fizzbuzz")
    elif((i%3)==False):
        print("fizz")
    elif not((i%5)):
        print("buzz")
    else:
        print(i)

File: program/fizzbuzz.js
//本プログラムの動作を確認する場合fizzbuzz_js.htmlを開いてください．
//fizzbuzz_js.htmlと同じフォルダに本ファイルをおいてください．
//fizzbuzz_js.htmlを開いた後右クリックを押し検証をクリックしてください．

//To check how this program works, please open fizzbuzz_js.html.
//Please place this file in the same folder as fizzbuzz_js.html.
//After opening fizzbuzz_js.html, right click and click "Inspect".


const button = document.getElementById("JSbutton");

button.addEventListener('click',function(){

    for(var i=0;i<100;i++){
        if((i%15)==0){console.log("fizzbuzz");}
        else if((i%3)==0){console.log("fizz");}
        else if((i%5)==0){console.log("buzz");}
        else {console.log(i);}
    }

    console.log("button clicked");
});